docs/content/grafana.md

```diff
@@ -26,6 +26,23 @@ spec:
 
 A folder simply has a name and a title.
 
+## Nested Folders
+Grizzly supports creation of nested folders, which can be useful when you don't want to have ALL your dashboards in a single one.
+To achieve that, you need to improve your DashboardFolder definition to add the uuid of the parent folder:
+
+```
+apiVersion: grizzly.grafana.com/v1alpha1
+kind: DashboardFolder
+metadata:
+  name: defaultnestedfolder
+spec:
+  parentUid: default
+  title: nestedfolder
+  uid: nesteddefaultfolder
+```
+
+This will create a new folder called `nestedfolder`with a uid: `nesteddefaultfolder`under the folder: `default`in your grafana instance.
```

Comment:
```suggestion
Grizzly supports the creation of nested folders.
To create one, specify the *uid* of the parent folder to a `DashboardFolder` definition:

```yaml
apiVersion: grizzly.grafana.com/v1alpha1
kind: DashboardFolder
metadata:
  name: nestedFolderSample
spec:
  parentUid: sample
  title: Nested folder
```

This will create a new folder "Nested folder" under the `sample` folder.
```

To keep the code sample linked to what the documentation was previously describing :)

Reply:
Looks like github doesn't like edit suggestions with nested backticks blocks :(